src/utils/random-utils.js

```diff
@@ -48,9 +48,9 @@ export const getSeedFromURL = () => {
 
 /**
  * Get an array of numbers
- * @param {number} n Length of the array
+ * @param {number} num Length of the array
```

Comment:
I think since the description is 'Length of the array' then maybe better name for this one is 'length'?